generic3g/actions/TimeInterpolateAction.F90

```diff
@@ -0,0 +1,121 @@
+#include "MAPL_Generic.h"
+
+module mapl3g_TimeInterpolateAction
+   use mapl3g_ExtensionAction
+   use mapl3g_regridder_mgr
+   use mapl3g_InfoUtilities
+   use MAPL_FieldUtils
+   use MAPL_Constants, only: MAPL_UNDEFINED_REAL
+   use mapl_ErrorHandling
+   use esmf
+
+   implicit none
+   private
+
+   public :: TimeInterpolateAction
+
+   type, extends(ExtensionAction) :: TimeInterpolateAction
+   contains
+      procedure :: initialize
+      procedure :: run
+   end type TimeInterpolateAction
+
+   interface TimeInterpolateAction
+      module procedure :: new_TimeInterpolateAction
+   end interface TimeInterpolateAction
+
+contains
+
+   function new_TimeInterpolateAction() result(action)
+      type(TimeInterpolateAction) :: action
+   end function new_TimeInterpolateAction
```

Comment:
```suggestion

contains

```

Reply:
If the constructor is a placeholder that will be updated, I understand. If not, I do not understand the purpose on a custom constructor.

Reply:
It was not obvious at the outset that the class would have no components. (Mixing programming language idioms.)   But there is a script to follow ... (Indeed, I cut-and-paste from a sibling subclass.) And this applies to the client code what constructs things.   It could leave someone wondering why some Action subclasses use a constructor and others just allocate an uninitialized object.

Having a blank constructor also insulates from changes that may creep into the parent class, so I'm inclined to leave this here for now.

Reply:
But I also admit that I find the compiler message "function result 'action' has not been given a value." to be rather annoying.